[HPRO-1143] Update README and other config for new installation of project (#1132)
all-of-us/healthpro: README.md

`npm run watch`

> Run docker container

Reviewer: ```suggestion
Run Docker containers
```

> `docker-compose up`

Reviewer: ```suggestion
`docker compose up`
```

(compose v1 is now [deprecated](https://www.docker.com/blog/announcing-compose-v2-general-availability/))

Configure your local development parameters by copying the `dev_config/config.yml.dist` file to `dev_config/config.yml`.  Edit `config.yml` as needed.  This file is .gitignore'd.  See comments in the `config.yml.dist` file for more details.

#### MySQL database configuration
Create a new MySQL for this application.  Configure the MySQL connection in `config.yml`. 

> Then, run `bin/console doctine:migrations:migrate` to build your local database.

Reviewer: Did you mean to leave this here? Not sure if this was accidental or if it's still a good idea to have a local database configured outside of Docker since we still run the deploy from the host machine.

Author: Yes, I left it intentionally since the deploy command runs the tests and needs local db access.

> Then, run `docker exec -it healthpro-web bin/console doctine:migrations:migrate` to build your database in docker container.

Reviewer: We have the `bin/dx` shortcut that I usually use. You could include that here, and maybe also an explanation of what it is.

```suggestion
Then, run `bin/dx bin/console doctrine:migrations:migrate` to execute the necessary Doctrine migrations.

(`bin/dx` is a convenience wrapper for executing a command inside the Docker web container. You can also run `bin/dx bash` to open a shell into the container.)
```
